test/general/sycl_iterator/sycl_iterator_for.pass.cpp

```diff
@@ -274,6 +303,20 @@ DEFINE_TEST(test_destroy_n)
 {
     DEFINE_TEST_CONSTRUCTOR(test_destroy_n)
 
+
+    template <typename Policy, typename Iterator1, typename Size>
+    void
+    operator()(Policy&&, __it_test_type<Iterator1>, __it_test_type<Iterator1>, Size)
```

Comment:
Usually in tests we have sometimes two `operator()` implementations with `std::enble_if_t`.
In this case - one will have
```C++
::std::enable_if_t<std::is_trivially_copyable_v<TestValueType>>
```
and the second simple `void`

Reply:
Actually, both  approach are correct.
Since we support C++14/17 `if contsexpr` also used construction in oneDPl code.

Reply:
The comment is not actual any more - the code was changed.